sklearn/utils/estimator_checks.py

```diff
@@ -4247,6 +4252,8 @@ def _output_from_fit_transform(transformer, name, X, df, y):
         data_fit,
         data_transform,
     ) in cases:
+        if not all(hasattr(transformer, meth) for meth in ["fit", "transform"]):
+            continue
```

Comment:
I know it adds 1 level of identation but this condition should be moved outside the loop